tests/internal/CMakeLists.txt

```diff
@@ -43,6 +43,7 @@ set(UNIT_TESTS_FILES
   processor.c
   uri.c
   msgpack_append_message.c
+  flb_conditionals.c
```

Comment:
flb_prefix shouldn't be needed for the test files.

conditionals is enough for the file name.